tendrl/commons/etcdobj/__init__.py

```diff
@@ -142,7 +155,7 @@ def __init__(self, **kwargs):  # pragma: no cover
         for key in dir(self):
             if not key.startswith('_'):
                 attr = getattr(self, key)
-                if issubclass(attr.__class__, Field):
+                if issubclass(attr.__class__, fields.Field):
```

Comment:
@r0h4n I was getting error here while testing, so I made this change. Please check if this is correct.